authenticator/src/test/java/com/amplifyframework/ui/authenticator/AuthenticatorViewModelTest.kt

```diff
@@ -57,6 +57,7 @@ class AuthenticatorViewModelTest {
 
     @Test
     fun `start only executes once`() = runTest {
+        coEvery { authProvider.fetchAuthSession() }.returns(AmplifyResult.Error(mockAuthException()))
```

Comment:
Use infix notation for mockk

```suggestion
        coEvery { authProvider.fetchAuthSession() } returns AmplifyResult.Error(mockAuthException())
```